libs/application/templates/financial-aid/src/fields/Confirmation/ApplicantConfirmation.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { ApproveOptions, FAFieldBaseProps } from '../../lib/types'
+import { confirmation } from '../../lib/messages'
+import { hasFiles, hasSpouse } from '../../lib/utils'
+import Confirmation from './Confirmation'
+
+const ApplicantConfirmation = ({ application }: FAFieldBaseProps) => {
+  const { answers, externalData } = application
+
+  const applicantHasSpouse = hasSpouse(answers, externalData)
+  const missingIncomeFiles =
+    answers.income === ApproveOptions.Yes && !hasFiles('incomeFiles', answers)
+
+  let firstStepText
+  switch (true) {
+    case (applicantHasSpouse && missingIncomeFiles):
+      firstStepText = confirmation.nextSteps.contentBothMissingFiles
+      break
+    case (applicantHasSpouse):
+      firstStepText = confirmation.nextSteps.contentSpouseMissingFiles
+      break
+    case (missingIncomeFiles):
+      firstStepText = confirmation.nextSteps.contentMissingFiles
+      break
+  }
```

Comment:
const firstStepText =  switch (true) {
    case (applicantHasSpouse && missingIncomeFiles):
      return confirmation.nextSteps.contentBothMissingFiles
     
Was more what I was thinking 🤔

Reply:
I don't think that syntax is allowed. The closest thing to this that I can come up with is:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/43727721/160649561-3b341ef1-4229-4573-a10b-8625e3e08361.png">

Reply:
yes it was just a pseudo code as an example, yea i would take out the break and have return and have not have firstStepText as string rather, I suggest having it as a function that return a string like in the example you made above 🙌🏼